packages/hoppscotch-sh-admin/src/components/setup/DataSharingAndNewsletter.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="flex flex-col items-center justify-center min-h-screen">
+    <div class="text-center">
+      <h2 class="text-lg">{{ t('data_sharing.welcome') }}</h2>
+      <img src="/assets/images/hoppscotch-title.svg" alt="" class="mt-2 w-56" />
+    </div>
+    <div
+      class="bg-primaryLight mt-10 p-10 border-2 border-dividerLight rounded-lg"
+    >
+      <section>
+        <div class="flex flex-col">
+          <p class="text-lg font-bold text-white">
+            {{ t('data_sharing.title') }}
+          </p>
+          <p class="text-sm font-light">
+            {{ t('data_sharing.description') }}
+          </p>
+          <HoppSmartToggle
+            :on="dataSharingToggle"
+            @change="dataSharingToggle = !dataSharingToggle"
+            class="my-5 text-white w-min justify-start"
+          >
+            {{ t('data_sharing.toggle_description') }}
+          </HoppSmartToggle>
+          <HoppButtonSecondary
+            outline
+            filled
+            :icon="IconShieldQuestion"
+            :label="t('data_sharing.see_shared')"
+            to="http://docs.hoppscotch.io"
+            class="w-min mx-auto mb-2"
+          />
+        </div>
+
+        <div class="mt-5">
+          <p class="text-lg font-bold text-white">
+            {{ t('newsletter.title') }}
+          </p>
+          <p>{{ t('newsletter.description') }}</p>
+          <HoppSmartToggle
+            :on="newsletterToggle"
+            @change="newsletterToggle = !newsletterToggle"
+            class="my-5 text-white"
+          >
+            {{ t('newsletter.toggle_description') }}
+          </HoppSmartToggle>
+        </div>
+      </section>
+
+      <div class="flex flex-col w-48 mx-auto mt-5">
+        <HoppButtonPrimary
+          :icon="IconLogIn"
+          :label="t('app.continue_to_dashboard')"
+          @click="submitSelection"
+        />
+        <a href="http://docs.hoppscotch.io">
+          <HoppButtonSecondary
+            outline
+            filled
+            :icon="IconBookOpenText"
+            :label="t('app.read_documentation')"
+            class="mt-3 px-5"
+          />
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
```

Comment:
```suggestion
<template>
  <div
    class="flex flex-col items-center justify-center min-h-screen space-y-10"
  >
    <div class="flex items-center justify-center flex-col space-y-2">
      <h2 class="text-lg">{{ t('data_sharing.welcome') }}</h2>
      <img src="/assets/images/hoppscotch-title.svg" alt="" class="w-52" />
    </div>
    <div
      class="bg-primaryLight p-10 border-2 border-dividerLight rounded-lg flex flex-col space-y-8"
    >
      <div class="flex flex-col space-y-5 items-start">
        <span>
          <p class="text-lg font-bold text-white">
            {{ t('data_sharing.title') }}
          </p>
          <p class="font-light">
            {{ t('data_sharing.description') }}
          </p>
        </span>
        <HoppSmartToggle
          :on="dataSharingToggle"
          @change="dataSharingToggle = !dataSharingToggle"
        >
          {{ t('data_sharing.toggle_description') }}
        </HoppSmartToggle>
        <HoppSmartAnchor
          blank
          to="http://docs.hoppscotch.io"
          :label="t('data_sharing.see_shared')"
          class="underline"
        />
      </div>

      <div class="flex flex-col items-start space-y-5">
        <span>
          <p class="text-lg font-bold text-white">
            {{ t('newsletter.title') }}
          </p>
          <p class="font-light">{{ t('newsletter.description') }}</p>
        </span>
        <HoppSmartToggle
          :on="newsletterToggle"
          @change="newsletterToggle = !newsletterToggle"
        >
          {{ t('newsletter.toggle_description') }}
        </HoppSmartToggle>
      </div>

      <div class="flex flex-col space-y-5">
        <HoppButtonPrimary
          :icon="IconLogIn"
          :label="t('app.continue_to_dashboard')"
          @click="submitSelection"
        />
        <HoppSmartAnchor
          blank
          to="http://docs.hoppscotch.io"
          :icon="IconBookOpenText"
          :label="t('app.read_documentation')"
        />
      </div>
    </div>
  </div>
</template>
```

modified the UI a little bit, used flex in some block and added open in new tab to the the links
@JoelJacobStephen can you check if this works

![image](https://github.com/hoppscotch/hoppscotch/assets/53208152/bbc67d6d-1bf6-4702-8450-146595e13c1a)

Reply:
@nivedin Looks good!!